? Makefile
? cover_db
? diff
? t-cover
? t-diff
? lib/POE/Component/Generic/Net/SSH2.pma
? t/Config.pm
Index: Changes
===================================================================
RCS file: /home/cvs/poco-generic/Changes,v
retrieving revision 1.3
diff -u -r1.3 Changes
--- Changes	11 Apr 2006 08:33:12 -0000	1.3
+++ Changes	12 Apr 2006 08:12:42 -0000
@@ -1,5 +1,14 @@
 Revision history for POE::Component::Generic.
 
+0.07 Wed Apr 12 2006
+    - Added documentation
+    - Added __callback_argument and __postback_arguement
+    - Use Scalar::Util::reftype instead of ref()
+    - Added __package_register
+    - Added PoCo::Generic::Net::SSH2->exec and ->cmd
+    - Fixed PoCo::Generic::Object->DESTROY
+    - Added test cases to improve test coverage
+
 0.06 Tue Apr 11 2006
     - Added PoCo::Generic::Net::SSH2
     - Added t/91_ssh2.t which tests the above
Index: README
===================================================================
RCS file: /home/cvs/poco-generic/README,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 README
--- README	7 Apr 2006 20:15:31 -0000	1.1.1.1
+++ README	12 Apr 2006 08:12:42 -0000
@@ -21,6 +21,12 @@
 	make test
 	make install
 
+It is possible to configure some of the test cases by passing arguments to
+Makefile.PL.
+
+    perl Makefile.PL --help
+
+
 The project needs your help.  If you're interested, please contact
 Philip Gwyn at gwyn-at-cpan.org.
 
Index: TODO
===================================================================
RCS file: /home/cvs/poco-generic/TODO,v
retrieving revision 1.2
diff -u -r1.2 TODO
